src/main/features/claimant-response/routes/payment-option.ts

```diff
@@ -75,7 +75,6 @@ export class PaymentOptionPage extends AbstractPaymentOptionPage<DraftClaimantRe
         }
         return courtOfferedPaymentIntention
       }
-      return undefined
```

Comment:
is this intentional ?

Reply:
Yes, it's unreachable now. The prior code in this block is `if (something) { ... return } else { ... return } line 78`

Reply:
yes, code never reaches there....i should have removed that as part of my previous PR. Noticed it now..